juju/juju
# enable-ha should ignore all constraints when given '--to' directive

## Description of change
enable-ha was incorrectly using constraints with '--to' directive

## QA steps
create a machine with tag 'foo' in MAAS, do juju bootstrap --bootstrap-constraints tags=foo, then do juju enable-ha --to machine2,machine3 , both without 'foo' tag, see if it works.

## Documentation changes
None

## Bug reference
https://bugs.launchpad.net/juju/+bug/1684049
